btr-api/sample_data/load_data.py

```diff
@@ -97,10 +100,12 @@ def _get_entity_stmnts():
                 'foundingDate': row['foundingDate'],
                 'dissolutionDate': row['dissolutionDate'],
             }
+            if limit and len(entity_stmnts.keys()) == limit:
```

Comment:
I usually just break the for loop if the next statement is return. When I have condition for testing. But this is good too.

Reply:
That's how I had it originally, but I thought this was clearer 😂